.github/scripts/ci_tests.sh

```diff
@@ -2,14 +2,19 @@
 
 # Copyright (c) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+# ci_tests.sh [-st] [-nxp] [-pc] [-esp]
+#
+# if none specified all vendors are build
 
 set -o errexit # Exit if command failed.
 set -o nounset # Exit if variable not set.
 set -o pipefail # Exit if pipe failed.
 
 TEST_SCRIPT_DIR=`dirname "$0"`
 TEST_FREERTOS_SRC=`pwd`/libs/FreeRTOS
-$TEST_SCRIPT_DIR/fetch_freertos.sh $TEST_FREERTOS_SRC
+RUN_BOARDS_BUILD=${@:-"-st -nxp -pc -esp"}
```

Comment:
$TEST_SCRIPT_DIR/fetch_freertos.sh $TEST_FREERTOS_SRC

this should not be removed.